src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.cs

```diff
@@ -96,7 +95,7 @@ public unsafe MetadataReader(byte* metadata, int length, MetadataReaderOptions o
             this.Block = new MemoryBlock(metadata, length);
 
             _options = options;
-            this.Utf8Decoder = utf8Decoder;
+            this.StringDecoder = utf8Decoder;
```

Comment:
At the time, I thought just in case we need to decode something other than UTF8 at a later point in time. That was the reasoning for naming it utf8Decoder. In case there's another one around. Theoretical, but I'm not sure I get the change in direction now.